### DISCUSSION TITLE
ILI9486 display produceding loud high frequency noise.

### DISCUSSION THREAD

**Discussion by hboyd2003** [Bodmer/TFT_eSPI#2516] [Q&A - General] [answered]
Hello, I am working on a project with a ILI9486 and a FT6236. The screen is fully functional, however, the display produces a headache inducing noise around 8708hz.  I have attempted at changing the SPI_FREQUENCY to no avail. Lastly I also tried out ILI9488 and ILI9481 which both worked but with some issues.

Using the ILI9488 driver the colors are correct except white appears as a lime green. When inverted black appears as a pure white. Finally it produces a much quieter noise at around 7770hz. 

Just like the ILI9488 driver white on the ILI9481 driver appeared as a lime green. Additionally everything was reversed, however, it **produced absolutely no noise**!

Is there anything I can do to remedy this? Seeing how different drivers affect the noise I believe it should be possible but I don't know where to start.

Board I am using: [Seeed XIAO ESP32-C3](https://wiki.seeedstudio.com/XIAO_ESP32C3_Getting_Started/)
Display purchase page: [AliExpress](https://www.aliexpress.us/item/3256804945595005.html?spm=a2g0o.order_detail.order_detail_item.3.e9baf19cwbR3Ub&gatewayAdapt=glo2usa&_randl_shipto=US)
Config file:
<details>
  <summary>Setup0.h</summary>

```
// See SetupX_Template.h for all options available
#define USER_SETUP_ID 0

#define ILI9486_DRIVER

#define TFT_INVERSION_OFF
#define TFT_BL 5 //3
#define TFT_BACKLIGHT_ON HIGH

// For NodeMCU - use pin numbers in the form PIN_Dx where Dx is the NodeMCU pin designation
#define TFT_CS   2 //0  // Chip select control pin D8
#define TFT_DC   3 //1 // Data Command control pin
#define TFT_RST  4 //2 // Reset pin (could connect to NodeMCU RST, see next line)

#define TFT_MOSI 10 //10
#define TFT_MISO 9 //9
#define TFT_SCLK 8 //8
//#define TFT_RST  -1  // Set TFT_RST to -1 if the display RESET is connected to NodeMCU RST or 3.3V


#define LOAD_GLCD   // Font 1. Original Adafruit 8 pixel font needs ~1820 bytes in FLASH
#define LOAD_FONT2  // Font 2. Small 16 pixel high font, needs ~3534 bytes in FLASH, 96 characters
#define LOAD_FONT4  // Font 4. Medium 26 pixel high font, needs ~5848 bytes in FLASH, 96 characters
#define LOAD_FONT6  // Font 6. Large 48 pixel font, needs ~2666 bytes in FLASH, only characters 1234567890:-.apm
#define LOAD_FONT7  // Font 7. 7 segment 48 pixel font, needs ~2438 bytes in FLASH, only characters 1234567890:.
#define LOAD_FONT8  // Font 8. Large 75 pixel font needs ~3256 bytes in FLASH, only characters 1234567890:-.
#define LOAD_GFXFF  // FreeFonts. Include access to the 48 Adafruit_GFX free fonts FF1 to FF48 and custom fonts

#define SMOOTH_FONT


// #define SPI_FREQUENCY  20000000
#define SPI_FREQUENCY  16000000
// #define SPI_FREQUENCY  40000000


//#define SPI_TOUCH_FREQUENCY  2500000


 #define SUPPORT_TRANSACTIONS
```
</details>

**Comment by Bodmer** [comment, marked as answer]
I have not encountered this problem.

Maybe comment out lines 24 to 31 in the display init code to see if the display registers default to sensible values.
https://github.com/Bodmer/TFT_eSPI/blob/master/TFT_Drivers/ILI9486_Init.h#L24

Alternatively contact the display vendor and ask for the initialisation sequence and adapt the init code accordingly.

**Reply by hboyd2003** [reply]
This worked. I also tested another one of the displays and it does not make the noise, so it is possible there is something wrong with this specific one. Thanks for the help!
